apps/trading-e2e/src/integration/markets.ts

```diff
@@ -88,5 +90,28 @@ describe('markets table', () => {
     cy.wait('@Market');
     cy.contains('SUSPENDED MARKET');
     cy.url().should('include', '/markets/market-1');
+
+    verifyMarketSummaryDisplayed('Suspended');
   });
 });
+
+function verifyMarketSummaryDisplayed(expectedMarketState: string) {
```

Comment:
suggestion (non blocking): If we aren't re using this function prefer to just keep it inline

Reply:
Done...I think 😄